src/index.html

```diff
@@ -10,13 +10,72 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="image_logo"
+        class="img"
+      >
+        <img
+          src="images/logo.png"
+          alt="Logo Image"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="apple"
```

Comment:
The `href` attribute for the 'apple' link should not contain spaces. Consider replacing spaces with hyphens, underscores, or encoding them as `%20` to ensure valid URL formatting.